src/class/cdc/cdc_device.h

```diff
@@ -241,6 +247,18 @@ static inline bool tud_cdc_write_clear(void)
   return tud_cdc_n_write_clear(0);
 }
 
+// Get the Receive FIFO
+static inline tu_fifo_t* tud_cdc_get_rx_ff (void)
+{
+  return tud_cdc_n_get_rx_ff(0);
+}
+
+// Get the transmit FIFO
+static inline tu_fifo_t* tud_cdc_get_tx_ff (void)
+{
+  return tud_cdc_n_get_tx_ff(0);
+}
+
```

Comment:
should be reverted since it has its own pr.

Reply:
Oops.. now it's better.